spec/lib/myfinance/resources/entity_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+
+describe Myfinance::Resources::Entity do
+  describe '.find_all', vcr: true do
+    context 'when success' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.